apps/communications/contacts/js/nfc.js

```diff
@@ -2,10 +2,9 @@
 
 /* global LazyLoader, ContactToVcard, MozNDEFRecord, fb, utils,
           NDEF, NfcUtils, utils*/
+/* exported NFC */
 
-var contacts = window.contacts || {};
-
-contacts.NFC = (function() {
+var NFC = (function() {
```

Comment:
Ensure that the namespace is set properly in [1]

[1] https://github.com/fcampo/gaia/blob/nfc_out_contacts_1173274/apps/communications/contacts/js/nfc.js#L83